checker-qual-android/build.gradle

```diff
@@ -0,0 +1,69 @@
+evaluationDependsOn(":checker-qual")
+
+task copySources(type: Copy) {
+    from files('../checker-qual/src/')
+    include "**/*.java"
+    into file('src')
+    fileMode(0555)
+}
+copySources.dependsOn(":checker-qual:copySources")
+copySources.dependsOn(":checker-qual:copyCompatQual")
+
+task replaceAnnotations {
+    doLast {
+        fileTree(dir: 'src', include: "**/*.java").each {
+            it.text = it.text.replaceAll("RetentionPolicy.RUNTIME", "RetentionPolicy.CLASS")
```

Comment:
Would you please add a comment explaining why `RUNTIME` is preferable to `CLASS` for Android?  

There's text in the issue that you could copy-paste:

>Types annotated with runtime annotations are always kept in the main dex by the default Android Gradle plugin. Using the standard Checker Framework annotations can lead to main dex overflows; users of the Checker framework may find themselves unable to build their Android apps. By contrast, class-retention annotations are stripped out before packaging by all build systems as a convention.